src/node/util.ts

```diff
@@ -413,7 +412,7 @@ export const isObject = <T extends object>(obj: T): obj is T => {
  * we don't have to set up a `vs` alias to be able to import with types (since
  * the alternative is to directly import from `out`).
  */
-const enum CharCode {
+enum CharCode {
```

Comment:
Playwright's test-runner uses babel-plugin-transform-typescript which does not support const enums. See [here](https://babeljs.io/docs/en/babel-plugin-transform-typescript#caveats).

Reply:
Awesome! 🎉